android/src/main/java/com/reactnativepagerview/PagerViewViewManager.kt

```diff
@@ -195,6 +205,13 @@ class PagerViewViewManager : ViewGroupManager<ViewPager2>() {
     }
   }
 
+  private fun refreshViewChildrenLayout(view: View) {
```

Comment:
please also call this method inside `setCurrentItem` method

Reply:
Done!